Cargo.toml

```diff
@@ -1,3 +1,7 @@
 [workspace]
 
-members = []
+members = [
+    "components/icu",
```

Comment:
So, icu and icu4x are going to stay dumb placeholders, and locale will have real implementation? We are just defining icu/icu4x here so we can take crates.io namespace.

Reply:
Yes. Eventually, I expect `icu` to start operating as an `umbrella` crate that bundles other crates into a single `I want to use ICU4X 1.0`, and `icu4x` will remain a dumb placeholder.